client/my-sites/email/email-management/gsuite-user-item/index.jsx

```diff
@@ -20,40 +20,67 @@ import './style.scss';
 function GSuiteUserItem( props ) {
 	const translate = useTranslate();
 	const [ dialogVisible, setDialogVisible ] = useState( false );
+
 	const onFixClickHandler = ( e ) => {
 		e.preventDefault();
 		setDialogVisible( true );
 	};
+
 	const onCloseClickHandler = () => {
 		setDialogVisible( false );
 	};
 
+	const getMailboxLink = () => {
+		const { email } = props.user;
+
+		return `https://accounts.google.com/AccountChooser?Email=${ email }&service=CPanel&continue=https://mail.google.com/`;
+	};
+
 	const getLoginLink = () => {
 		const { email, domain } = props.user;
+
 		return `https://accounts.google.com/AccountChooser?Email=${ email }&service=CPanel&continue=https://admin.google.com/a/${ domain }`;
 	};
 
+	const renderMailbox = () => {
+		return (
+			<ExternalLink
+				icon
+				href={ getMailboxLink() }
+				onClick={ props.onClick }
+				target="_blank"
+				rel="noopener noreferrer"
+			>
+				{ translate( 'Mailbox', { context: 'Link pointing to Gmail' } ) }
+			</ExternalLink>
+		);
+	};
+
 	const renderManage = () => {
+		if ( ! props.user.is_admin ) {
+			return;
+		}
+
 		return (
 			<ExternalLink
 				icon
-				className="gsuite-user-item"
 				href={ getLoginLink() }
 				onClick={ props.onClick }
 				target="_blank"
 				rel="noopener noreferrer"
 			>
-				{ translate( 'Manage', { context: 'Login to G Suite Manage' } ) }
+				{ translate( 'Manage', { context: 'Link pointing to Google Admin Console' } ) }
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **24** times:
`translate( 'Manage', { context: 'Login to G Suite Manage'} )` *ES Score: 11*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/gsuite-finish-setup-notices)